core/config/v2/docs/secrets.toml

```diff
@@ -45,3 +45,17 @@ AuthToken = "pyroscope-token" # Example
 #
 # Environment variable: `CL_PROMETHEUS_AUTH_TOKEN`
 AuthToken = "prometheus-token" # Example
+
+[Mercury]
+# ID is the Mercury ID assigned for this Automation node
+#
+# Environment variable: `CL_MERCURY_ID`
+ID = "A-Mercury-ID" # Example
+# Key is the Mercury Key assigned for this Automation node
+#
+# Environment variable: `CL_MERCURY_KEY`
+Key = "A-Mercury-Key" # Example
+# URL is the Mercury endpoint URL which this Automation node will query in its MercuryLookup process
+#
+# Environment variable: `CL_MERCURY_URL`
+URL = "A-Mercury-URL" # Example
```

Comment:
better to make this multiple, like example here: https://github.com/smartcontractkit/chainlink/pull/8340/files#diff-423d8ff32c162c55aeee40cfd114daac935f5b3a95441b4524548930a077764a

Reply:
It should be possible to support multiple automation jobs on the same node for different mercury servers.